### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by benharri**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# #! book
+
+A FOSS hackers guide to the CLI, privacy, security, self hosting, and the internet.
```
Comment (RyanSquared) [inline README.md:3]: technically not "FOSS" as it's not software

Comment (RyanSquared) [reply README.md:3]: also not yet licensed so i'm not sure how "free" it is

Comment (benharri) [reply README.md:3]: That's fair. I just created it from the repo description. I'd like to have a ToC on the README eventually.

Comment (daurnimator) [reply README.md:3]: I read "FOSS hacker" as one thing.

Comment (RyanSquared) [reply README.md:3]: That's fair. I guess I didn't put that together mentally.

**Comment by RyanSquared** [comment]
@benharri do you mind if I instead push a change onto #3 including an autogenerated table of contents?

**Comment by benharri** [comment]
@RyanSquared  feel free. this was not anything that needs to be saved.

**Comment by RyanSquared** [comment]
cool. mind reviewing over on #3 ? i pushed what i had now, which is a somewhat working Python file.

**Pull request closed without merging**